app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('axios'),
+  constants = require('../constants'),
+  errors = require('../errors'),
+  logger = require('../logger'),
+  photosService = require('./photos');
+
+const { ALBUM_API_URL } = constants;
+
+exports.getAlbum = async id => {
+  const url = `${ALBUM_API_URL}/albums?id=${id}`;
+  logger.info(`Making a request to url ${url}`);
+  try {
+    const albumInfo = await axios.get(`${url}`);
+    if (!albumInfo.data.lenght) {
+      throw errors.notFound(`Album with id ${id} not found`);
+    }
+    const photosForAlbum = await photosService.getPhotosForAlbum(id);
+    return {
+      id: albumInfo.data[0].id,
```

Comment:
save `albumInfo.data[0]` in a variable with a shorter name to make code more concise and to avoid indexing many times